src/libpmemfile-posix/lseek.c

```diff
@@ -191,7 +191,8 @@ lseek_seek_data_or_hole(PMEMfilepool *pfp, struct pmemfile_vinode *vinode,
 	if (vinode_is_regular_file(vinode)) {
 		if (whence == PMEMFILE_SEEK_DATA) {
 			offset = lseek_seek_data(pfp, vinode, offset, fsize);
-			ASSERT(offset <= fsize);
+			if (offset > fsize)
```

Comment:
I'm not sure if this should be `offset > fsize` or `offset >= fsize`.
E.g. what should happen with an empty file, should offset == 0 be OK with SEEK_DATA?